TRIAGING.md

```diff
@@ -0,0 +1,89 @@
+
+Meetings are hosted regularly at 8 AM Pacific Time on Tuesdays bi-weekly and can be joined via the links posted on the 
+[OpenSearch Meetup Group](https://www.meetup.com/opensearch/events/) list of events. The event will be titled 
+`Development Backlog & Triage Meeting - k-NN/neural-search`.
+
+After joining the Chime meeting, you can enable your video / voice to join the discussion.  If you do not have a webcam 
+or microphone available, you can still join in via the text chat.
+
+If you have an issue you'd like to bring forth please consider getting a link to the issue so it can be presented to 
+everyone in the meeting.
+
+### Is there an agenda for each week?
+
+Meetings are 60 minutes and structured as follows:
+
+1. Initial Gathering: As we gather, feel free to turn on video and engage in informal and open-to-all conversation.  After a bit a volunteer will share their screen and proceed with the agenda.
+2. Announcements: If there are any announcements to be made they will happen at the start of the meeting.
+3. Review of New Issues: The meetings always start with reviewing all untriaged/recent issues for the k-NN and neural-search repositories.
+4. Member Requests: Opportunity for any meeting member to ask for consideration of an issue or pull request.
+5. Pull Request Discussion: Then, we review the status of outstanding pull requests from the k-NN and neural-search repositories.
+6. Open Discussion: Allow for members of the meeting to surface any topics without issues filed or pull request created.
+
+
+There is no specific ordering within each category.
+
+If you have an issue you would like to discuss but do not have the ability to attend the entire meeting please attend when is best for you and signal that you have an issue to discuss when you arrive.
+
+### Do I need to have already contributed to the project to attend a triage meeting?
+
+No, all are welcome and encouraged to attend. Attending the Backlog & Triage meetings is a great way for a new contributor to learn about the project as well as explore different avenues of contribution.
+
+### What if I have an issue that is almost a duplicate, should I open a new one to be triaged?
+
+You can always open an issue including one that you think may be a duplicate. However, in cases where you believe there 
+is an important distinction to be made between an existing issue and your newly created one, you are encouraged to 
+attend the triaging meeting to explain.
+
+### What if I have follow-up questions on an issue?
+
+If you have an existing issue you would like to discuss, you can always comment on the issue itself. Alternatively, you 
+are welcome to come to the triage meeting to discuss.
+
+### Is this meeting a good place to get help setting up k-NN/neural-search features on my OpenSearch instance?
+
+While we are always happy to help the community, the best resource for implementation questions is [the OpenSearch forum](https://forum.opensearch.org/c/plugins/k-nn/48).
```

Comment:
The link to forum (https://forum.opensearch.org/c/plugins/k-nn/48) is pointing to k-nn. Do you know if we can create a topic for semantic search/neural search?

Reply:
@martin-gaievski Going to figure this out asynchronously. For now, will leave as is for k-NN.